meta-evals/correctness/test_correctness_evaluator.py

```diff
@@ -0,0 +1,138 @@
+# Tests for the criteria evaluator
+from typing import Optional
+from uuid import uuid4
+
+import pytest
+from langchain.callbacks.manager import CallbackManager
+from langchain.evaluation import load_evaluator
+from langchain.load import dump as langchain_dump
+from langchain.schema import runnable
+from langchain.smith import RunEvalConfig
+from langsmith import Client, EvaluationResult
+from langsmith.evaluation import RunEvaluator
+from langsmith.schemas import Example, Run
+
+
+class ExactScoreMatch(RunEvaluator):
+    def evaluate_run(self, run: Run, example: Example) -> EvaluationResult:
+        predicted_score = run.outputs["score"]
+        return EvaluationResult(
+            key="exact_score_match",
+            score=predicted_score == example.outputs["output_correctness_score"],
+        )
+
+
+@pytest.fixture(scope="session")
+def uid() -> str:
+    return uuid4().hex[:8]
+
+
+class EvaluatorRunnable(runnable.Runnable):
```

Comment:
This would be avoidable if runnable lambdas would optionally pass the callback manager down to the function